l2meter.gemspec

```diff
@@ -10,10 +10,4 @@ Gem::Specification.new do |spec|
   spec.license      = "MIT"
   spec.files        = Dir["LICENSE.txt", "README.md", "lib/**/**"]
   spec.require_path = "lib"
-
-  spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "pry-byebug"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec", "~> 3.8.0"
-  spec.add_development_dependency "timecop"
```

Comment:
Any specific reason to pull these back out of here, rather than sticking with the recommendation of RubyGems and keeping them here? https://github.com/heroku/l2meter/commit/eb0cf41f4fa484e34e353b3fb297126b148abdcd#commitcomment-33404223

Reply:
Oh, I've just reverted the commit to make the CI green. This commit has nothing to do with the intent of the PR

Reply:
👍 I think the bundler failures are caused by running on the older Travis infra. I've upgraded other Gems to the Xenial image and it ✨magically works ✨. Though I'm still unsure why. Need to dig into that another day.